apps/testrunner/test_suites/selenium_tests/main_page_selenium_test.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+from selenium.webdriver.common.keys import Keys
+from common.mw_url_generator import MWUrlGenerator
+from testrunner.test_suites.selenium_tests.selenium_test import SeleniumTest
+
+logger = logging.getLogger(__name__)
+
+class MainPageSeleniumTest(SeleniumTest):
+    def do_run(self, driver, timer):
+        self.get_and_measure(self.baseUrl, driver, timer)
```

Comment:
I would prefer different name. Perhaps measure() or get_measurements() something like that, 'and' in a function name sounds weird to me.